e2e-tests/micronaut-rest-api/pom.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+<!--    <groupId>com.foo.micronaut.rest</groupId>-->
+    <artifactId>micronaut-rest-api</artifactId>
+    <packaging>jar</packaging>
+
+    <parent>
+        <groupId>org.evomaster</groupId>
+        <artifactId>evomaster-e2e-tests</artifactId>
+        <version>1.3.1-SNAPSHOT</version>
+    </parent>
+
+    <properties>
+        <packaging>jar</packaging>
+        <jdk.version>11</jdk.version>
+        <release.version>11</release.version>
+        <exec.mainClass>com.foo.micronaut.rest.MicronautApplication</exec.mainClass>
+        <micronaut.runtime>netty</micronaut.runtime>
+        <micronaut.data.version>1.0.2</micronaut.data.version>
+        <micronaut.version>3.2.3</micronaut.version>
+        <micronaut-openapi.version>3.2.0</micronaut-openapi.version>
+    </properties>
```

Comment:
these versions should be in the root pom file, eg as we do for Spring, as those will be updated with new releases of Micronaut. in the case of patio module, we use an old version on purpose, which will override the root pom file.